pkgs/tools/networking/wireguard-go/update.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl gnused common-updater-scripts vgo2nix
+
+set -eu -o pipefail
+
+readonly BASEDIR="$(dirname $(readlink -f $0))"
+
+version="$(curl -sL https://build.wireguard.com/distros.txt | sed -n 's/^upstream\tgo\t\([^\t]\+\)\t.*/\1/p')"
+update-source-version wireguard-go "$version"
+
+vgo2nix -dir $(nix-build -A wireguard-go.src) -outfile $PWD/pkgs/tools/networking/wireguard-go/deps.nix
```

Comment:
I'm not sure if it's right to assume `$PWD` to be like this, or if there's a better way to do it -  I couldn't find a good example for that. /cc @jtojnar

Reply:
Currently, this is undefined, see https://github.com/NixOS/nixpkgs/issues/61935, using `git rev-parse --show-toplevel` is probably more safe bet.

Reply:
Updated it to use that, thanks!